c4200m_get_target_sentences.py

```diff
@@ -5,44 +5,75 @@
 
 from absl import app
 import tensorflow_datasets as tfds
+import multiprocessing as mp
+import re
 
 LOGGING_STEPS = 100000
+BATCH_SIZE = 64
 
+def rreplace(string, old, new):
+  return new.join(string.rsplit(old, 1))
+
+def get_file_paths(edits_tsv_path, concurrent_runs):
+  start_index_string = re.search('(\d+)', edits_tsv_path).group(0)
```

Comment:
This could go wrong if the edit_tsv_path contains a number somewhere else.

Reply:
thanks for the tip! let me fix this!